migrations/sqlite/20240603102429_access_tokens_table.sql

```diff
@@ -8,4 +10,8 @@ CREATE TABLE access_tokens(
     account_id VARCHAR(100) NOT NULL REFERENCES accounts(id)
 );
 
-CREATE UNIQUE INDEX idx_account_token_name ON access_tokens(account_id, token_name);
\ No newline at end of file
+CREATE UNIQUE INDEX idx_access_tokens_account_token_name 
+    ON access_tokens(account_id, token_name) 
+    WHERE revoked_at IS NULL;
```

Comment:
Since I'm making a breaking change in flow configurations storage anyway, I decided to simplify the migration files, so that the schema code is more readable. Europort will be reset from scratch, for Demo the database does not exist yet.